yt/fields/field_info_container.py

```diff
@@ -144,10 +145,16 @@ def setup_particle_fields(self, ptype, ftype="gas", num_neighbors=64):
                 raise RuntimeError
             if field[0] not in self.ds.particle_types:
                 continue
+            units = self.ds.field_units.get(field, None)
+            if units is None:
+                try:
+                    units = ytcfg.get("fields", *field, "units")
+                except KeyError:
+                    units = ""
```

Comment:
I forget. Is this a valid way to express "dimensionless" ? If so, isn't there another, more explicit way ?

Reply:
It's valid